include/envoy/grpc/google_grpc_creds.h

```diff
@@ -39,6 +39,8 @@ class GoogleGrpcCredentialsFactory {
    * a Google gRPC credentials factory.
    */
   virtual std::string name() const PURE;
+
+  static std::string category() { return "grpc_credentials"; }
```

Comment:
comment on all method in include headers.

Reply:
Fixed.